src/modules/fundingManager/bondingCurveFundingManager/BancorVirtualSupplyBondingCurveFundingManager.sol

```diff
@@ -275,37 +275,64 @@ contract BancorVirtualSupplyBondingCurveFundingManager is
     /// @inheritdoc IBancorVirtualSupplyBondingCurveFundingManager
     function calculatePurchaseReturn(uint _depositAmount)
         external
-        view
         returns (uint mintAmount)
     {
         if (_depositAmount == 0) {
             revert
                 BancorVirtualSupplyBondingCurveFundingManager__InvalidDepositAmount(
             );
         }
-        if (buyFee > 0) {
```

Comment:
You could do the zero check instead in the _calculateNetAmountAndFee:
if(_feePct==0){
return _transactionAmount}
.....